framework/i18n/Formatter.php

```diff
@@ -787,6 +787,50 @@ public function asRelativeTime($value, $referenceTime = null)
         }
     }
 
+    /**
+     * Formats the value as a time interval.
+     * @param mixed $value the value to be formatted, in ISO8601 format.
```

Comment:
It could be detected if it's a valid ISO format or a number. But for numbers there should be an optional argument with some kind of denominator if somebody uses a base unit different than second (for example, millisecond).